src/Entities/Player.cpp

```diff
@@ -364,7 +364,7 @@ void cPlayer::TickFreezeCode()
 	}
 	else
 	{
-		if (!GetClientHandle()->IsPlayerChunkSent())
+		if (!GetClientHandle()->IsPlayerChunkSent() || (!GetParentChunk()->IsValid()))
```

Comment:
I would expect that `!GetClientHandle()->IsPlayerChunkSent()` implies `(!GetParentChunk()->IsValid()`, but it is not the case.